neuralforecast/common/_base_recurrent.py

```diff
@@ -48,6 +54,16 @@ def __init__(
         # Loss
         self.loss = loss
         self.learning_rate = learning_rate
+        self.max_steps = max_steps
+        self.num_lr_decays = num_lr_decays
+        self.lr_decay_steps = (
+            max(max_steps // self.num_lr_decays, 1)
+            if self.num_lr_decays > 0
+            else max_steps
+        )
+        self.early_stop_patience_steps = early_stop_patience_steps
```

Comment:
What happens with `EarlyStopping(patience=-1)`
Should we overwrite `early_stop_patience_steps` when it is negative?

Reply:
Nothing, see if below in definition of callbacks